packages/bot-avs/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "@wireapp/bot-avs",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "GPL-3.0",
+  "dependencies": {
+    "@wireapp/avs": "6.1.9",
+    "@wireapp/core": "16.10.9",
+    "dotenv": "8.2.0",
+    "wrtc": "0.4.5"
+  },
+  "scripts": {
+    "start": "cross-env NODE_DEBUG=@wireapp* ts-node src/main.ts"
+  },
+  "devDependencies": {
+    "@types/webrtc": "0.0.26"
```

Comment:
AWS will only install main dependencies (no dev dependencies) and run `yarn start` by default. The start script uses `ts-node` and `ts-node` will fail to compile if it cannot find types for the `wrtc` package so `@types/webrtc` has to become a first-class `dependency`.